### PR TITLE
Added a-la-carte simple guide with list of component and automatic generator ready to easily copy the selected elements

### PR DESCRIPTION
**Pull request by adriaroms**
Essentially, this is the most important part:
screen shot 2017-12-06 at 21 38 27

It is a list in which you can search by name of the markup or by name of the import, and you can select with the checkbox the components you want, and to end if we click on the button with the copy icon in the bottom left, We copy the formatted components ready to paste in our project.

Adrià.

### CODE REVIEW

File: i18n/en-US/guides/ALaCarte.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+export default {
+  header: 'Overview',
```
Comment (johnleider) [inline i18n/en-US/guides/ALaCarte.js]: convert all lines to use markdown

File: pages/guides/ALaCartePage.vue [hunk 2]
```diff
@@ -1,4 +1,160 @@
 <template lang="pug">
   doc-view
-    div.display-2 Under construction
+    input(:style="{ position: 'absolute', left: '-1000px', top: '-1000px' }" :value="copy" ref="copy")
```
Comment (johnleider) [inline pages/guides/ALaCartePage.vue]: any element with more than 2 properties should have them broken into new lines

File: pages/guides/ALaCartePage.vue [hunk 3]
```diff
@@ -1,4 +1,160 @@
...
+       | });
+
+    section#component-name-list
+      section-head(value="Guides.ALaCarte.componentNameListHeader")
+      section-text(value="Guides.ALaCarte.componentNameListText1")
+      v-card
+        v-card-title
+          v-spacer
+          v-text-field(append-icon="search" label="Search" single-line hide-details v-model="search")
+        v-data-table(v-bind:headers="headers" v-bind:items="items" :customFilter="customFilter" v-bind:search="search" v-model="selected" item-key="name" select-all class="elevation-1")
```
Comment (johnleider) [inline pages/guides/ALaCartePage.vue]: when this goes into new lines, remove v-bind

File: pages/guides/ALaCartePage.vue [hunk 4]
```diff
@@ -1,4 +1,160 @@
...
+        { value: false, name: 'v-time-picker', component: 'VTimePicker', group: 'VTimePicker' },
+        { value: false, name: 'v-toolbar', component: 'VToolbar', group: 'VToolbar' },
+        { value: false, name: 'v-toolbar-side-icon', component: 'VToolbar', group: 'VToolbar' },
+        { value: false, name: 'v-tooltip', component: 'VTooltip', group: 'VTooltip' },
+        { value: false, name: 'transitions', component: 'transitions', group: 'transitions' },
+      ],
+    };
+  },
+  computed: {
+    copy() { return this.generateCustomComponent(); },
```
Comment (johnleider) [inline pages/guides/ALaCartePage.vue]: make this 3 lines

Comment (adriaroms) [reply pages/guides/ALaCartePage.vue]: ok.

File: pages/guides/ALaCartePage.vue (hunk 3 above)
Comment (adriaroms) [reply pages/guides/ALaCartePage.vue]: ok.

File: pages/guides/ALaCartePage.vue (hunk 2 above)
Comment (adriaroms) [reply pages/guides/ALaCartePage.vue]: ok.

File: i18n/en-US/guides/ALaCarte.js (hunk 1 above)
Comment (adriaroms) [reply i18n/en-US/guides/ALaCarte.js]: ok.

**Comment by adriaroms** [comment]
Changes requested by @johnleider applied.

**Pull request merged**